frontend/src/pages/instance/components/EditorLink.vue

```diff
@@ -89,7 +89,15 @@ export default {
             if (this.disabled) {
                 return false
             }
-            window.open(this.url, !this.isImmersiveEditor ? '_blank' : '_self')
+            let href = this.url
```

Comment:
This is a viable solution but for only one specific use case.

My take on this problem would be in the `frontend/src/ui-components/components/Button.vue` component, to completely remove the button and used an anchor stylized as a button or switch the type='anchor' where needed on the said component. That way we'd let the default html events take their course and we wouldn't need to intervene for niche cases

Reply:
Yes, but the issue is the URL is different depending on accessing the editor direct or immersive. As it stands, we have no option but to have a click handler for the descision stuff and a href for the middle click.

Reply:
my bad, my solution was focused on other places where this is happening!